modules/gdnative/nativescript/nativescript.cpp

```diff
@@ -957,6 +957,7 @@ void NativeScriptLanguage::_unload_stuff(bool p_reload) {
 
 NativeScriptLanguage::NativeScriptLanguage() {
 	NativeScriptLanguage::singleton = this;
+	has_objects_to_register = false;
```

Comment:
This should have been in the `#ifndef`, it breaks javascript compilation: https://travis-ci.org/GodotBuilder/godot-builds/jobs/347994030

Reply:
Fixed with 08cadc3d871f681ef1f951894345359a85ba190e.